src/hotspot/share/memory/arena.cpp

```diff
@@ -40,19 +40,19 @@
 // It is used very early in the vm initialization, in allocation
 // code and other areas.  For many calls, the current thread has not
 // been created so we cannot use Mutex.
-static PlatformMutex* _global_chunk_pool_mutex = nullptr;
+static PlatformMutex* GlobalChunkPoolMutex = nullptr;
```

Comment:
That naming style also seems consistent with some of the code base and with the style guide.  So okay.